### ISSUE TITLE
ini for configuring Broken Gear Drops Components 

### ISSUE THREAD

**Issue by Bluberry-Kat** [FalsehoodMC/Fabrication#19]
presuming this is a hardcoded list right now
would be nice to have this configured for a proper modded context, like Tech Reborn's gem tools 

the config gui specifically mentions not wanting to use datapacks, but maybe it could use the system you've written for the mod already?

**Comment by exaskye** [comment]
It is a [particularly spicy variety of hardcoded](https://github.com/unascribed/Fabrication/blob/c058a5a/src/main/java/com/unascribed/fabrication/mixin/f_balance/broken_tools_drop_components/MixinLivingEntity.java#L68-L174).

Would need to rewrite basically the whole thing for an INI-based list rather than this half-assed autodetection. Probably a good idea in general.

**Comment by exaskye** [comment]
Thinking about syntax...

```ini
[@options]
drop_rate=75
guaranteed_ingots=1

[@materials.minecraft:]
diamond=coal=1/4 diamond=1 diamond_block=9
gold=gold_nugget=1/9 gold_ingot=1 gold_block=9
netherite=netherite_scrap=1/4
stick=stick=1

[minecraft:]
diamond_pickaxe=3xdiamond 2xstick
netherite_boots=4xdiamond 4xgold netherite!
```

I think most of this is self-explanatory, though the ! on netherite means "exempt from drop_rate", and the lack of a quantity specifier means it drops one.

**Comment by exaskye** [comment]
Closed by 58d1bc209de2b93994f2c9c5c198f5fcf45614ff
